src/Symfony/Component/HttpKernel/CacheWarmer/CacheWarmerAggregate.php

```diff
@@ -106,6 +108,11 @@ public function warmUp(string $cacheDir)
                 file_put_contents($this->deprecationLogsFilepath, serialize(array_values($collectedLogs)));
             }
         }
+
+        $preload = array_merge([], ...$preload);
+        $preload = array_keys(array_flip($preload));
+
+        return $preload;
```

Comment:
why not use `array_values` here ?

```suggestion
        return array_values($preload);
```

Reply:
to remove possible duplicates

Reply:
:+1: `array_values(array_unique($preload));` or just `array_unique($preload);` would have been clearer tho :stuck_out_tongue:

Reply:
but slower, unless `array_unique()` uses has maps internally?